tasks/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN dnf -y update && \
         git-lfs \
         gnupg \
         gobject-introspection-devel \
+        golang-github-evanw-esbuild \
```

Comment:
So this won't help me much, as it's installed a very old version of the bundler which we can't use. We need esbuid >= 17.7 which is not available in fedora. The package there was not updated with esbuild version bump since almost a year.

Reply:
And we need the latest crack? Well, this was purely opportunistic to play around with that, and compare "OS package" vs. "bundled". No commitment to anything right now.